src/main/webapp/app/shared/notification/notification-sidebar/notification-sidebar.component.ts

```
@@ -207,11 +214,12 @@ export class NotificationSidebarComponent implements OnInit, OnDestroy {
         if (!this.sortedNotifications) {
             this.recentNotificationCount = 0;
         } else if (this.lastNotificationRead) {
-            this.recentNotificationCount = this.sortedNotifications.filter((notification) => {
-                return notification.notificationDate && notification.notificationDate.isAfter(this.lastNotificationRead!);
-            }).length;
+            this.recentNotificationCount =
+                this.sortedNotifications.filter((notification) => {
+                    return notification.notificationDate && notification.notificationDate.isAfter(this.lastNotificationRead!);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)